packages/core/src/components/graphs/area-stacked.ts

```diff
@@ -67,11 +68,20 @@ export class StackedArea extends Component {
 		areas.exit().attr("opacity", 0).remove();
 
 		const enteringAreas = areas.enter().append("path").attr("opacity", 0);
+		const userProvidedScale = Tools.getProperty(options, "color", "scale");
+		const noProvidedColorScale = userProvidedScale === null || Object.keys(userProvidedScale).length === 0;
 
 		enteringAreas
 			.merge(areas)
 			.data(stackedData, (d) => d[0][groupMapsTo])
-			.attr("fill", (d) => self.model.getFillColor(d[0][groupMapsTo]))
+			.attr("class", "area")
+			.attr(
+				"class",
+				(d) => noProvidedColorScale
+				? `area ${this.model.getColorClass(d[0][groupMapsTo])}`
+				: "area"
+			)
+			.attr("fill", (d) => noProvidedColorScale ? null : self.model.getFillColor(d[0][groupMapsTo]))
```

Comment:
Only `model` should be looking at `Tools.getProperty(options, "color", "scale")`